packages/office-ui-fabric-react/src/components/ComboBox/ComboBox.tsx

```diff
@@ -1849,14 +1849,6 @@ export class ComboBox extends BaseComponent<IComboBoxProps, IComboBoxState> {
           return;
         }
 
-        // If we are not allowing freeform and
-        // allowing autoComplete, handle the input here
-        // since we have marked the input as readonly
-        if (!allowFreeform && autoComplete === 'on') {
-          this._onInputChange(String.fromCharCode(ev.which));
```

Comment:
>ev.which [](start = 50, length = 8)

why couldn't this be updated to `ev.key`?

Reply:
hmmm...for some reason i thought ev.key wasn't available on mouse down

Reply:
ok...reduced the change of this PR to simply using ev.key instead.